settings.default.php

```diff
@@ -23,6 +23,10 @@
 	'clean_idle_peers'  => 10,           /* tweaks % of time tracker attempts idle peer removal */
 	                                     /* if you have a busy tracker, you may adjust this */
 	                                     /* example: 1 = 1%, 10 = 10%, 50 = 50%, 100 = every time */
+	'honor_xff'        => false,          /* If this server is behind a frontend proxy, the client IP */
```

Comment:
I will come back to this once we have resolved if we should trust `ip=` and `ipv6=` over all else, in which case this is something we could have enabled by default.